internal/request/graphql/schema/generate.go

```diff
@@ -566,6 +566,20 @@ func (g *Generator) buildMutationInputTypes(collections []client.CollectionDefin
 					continue
 				}
 
+				if field.Kind == client.FieldKind_DocID && strings.HasSuffix(field.Name, request.RelatedObjectID) {
```

Comment:
thought: the logic here and in `client.Document` is the same. Would it make sense to have a `field.IsSecondaryRelation` helper function instead?

Reply:
I thought about it, but decided it wasn't worth coupling them together and boost the (public) client package surface area.  Especially given that the gql/schema package is begging for a re-write.